modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Event.scala

```diff
@@ -1,40 +1,42 @@
 package edu.gemini.seqexec.engine
 
-object Event {
-  /**
-    * Anything that can go through the Event Queue.
-    */
-  sealed trait Event
-  case class EventUser(ue: UserEvent) extends Event
-  case class EventSystem(se: SystemEvent) extends Event
+/**
+  * Anything that can go through the Event Queue.
+  */
+sealed trait Event
+case class EventUser(ue: UserEvent) extends Event
```

Comment:
`Event_` acts as a data type constructor wrapper whereas `_Event` is a concrete type. How is this usually written in Scala?